batch/Makefile

```diff
@@ -1,9 +1,26 @@
 .PHONY: build build-test \
   push push-test \
   run-docker run \
-  test test-local deploy
+  test test-local deploy clean
 
 PROJECT = $(shell gcloud config get-value project)
+PY_FILES = $(shell find batch -iname \*.py -not -exec git check-ignore -q {} \; -print)
```

Comment:
Did you still want the `-print` in there or was that for debugging only?

Reply:
With `-exec` but without `-print`, `find` does nothing:

```
(base) dking@wmb16-359 # find batch -iname \*.py -not -exec git check-ignore -q {} \;
(base) dking@wmb16-359 # find batch -iname \*.py -not -exec git check-ignore -q {} \; -print
batch/batch/__init__.py
batch/batch/api.py
batch/batch/client.py
batch/batch/requests_helper.py
batch/batch/server/__init__.py
batch/batch/server/globals.py
batch/batch/server/server.py
batch/setup.py
batch/test/__init__.py
batch/test/test_batch.py
```

`man find` documents this behavior (I was unaware until I wrote this PR):
```
     -print  This primary always evaluates to true.  It prints the pathname of the 
             current file to standard output.  If none of -exec, -ls, -print, -print0,
             or -ok is specified, the given expression shall be effectively replaced
             by ( given expression ) -print.
```